projects/ngx-tour-core/src/lib/tour.service.ts

```diff
@@ -37,6 +42,8 @@ export class TourService<T extends IStepOption = IStepOption> {
   public resume$: Subject<T> = new Subject();
   public anchorRegister$: Subject<string> = new Subject();
   public anchorUnregister$: Subject<string> = new Subject();
+  public next$: Subject<T> = new Subject();
```

Comment:
Might be a good idea to make only observable public and not the subject itself (`mySubject.asObservable()`).